addons/mail/static/src/core/common/message.xml

```diff
@@ -141,6 +151,9 @@
                     <i class="fa fa-lg" t-att-class="action.icon"/>
                 </button>
             </t>
+            <t t-set="TRANSLATE">Translate</t>
+            <t t-set="REVERT">Revert</t>
+            <button t-if="translatable" class="btn px-1 py-0 rounded-0" t-on-click="onClickToggleTranslation" tabindex="0" t-att-title="message.showTranslation ? REVERT : TRANSLATE" t-att-aria-label="message.showTranslation ? REVERT : TRANSLATE"><i class="fa fa-lg fa-language" t-att-class="message.showTranslation ? 'o-mail-Message-translated' : ''"/></button>
```

Comment:
Should define the action as a `message_actions`, so that it's not longer in template but automatically added according to its definition and other actions. In particular, it can set a sequence number that complies with other actions.

Reply:
Moved this button to the message actions as requested.